Beeper/barcelona-mautrix/BarcelonaMautrix.swift

```diff
@@ -57,8 +51,38 @@ class BarcelonaMautrix {
         } else {
             CLInfo("CoreSentry", "Starting without setting up Sentry")
         }
+    }
+    
+    static func getUnixSocketPath() -> String? {
+        guard let index = ProcessInfo.processInfo.arguments.firstIndex(of: "--unix-socket"),
+            ProcessInfo.processInfo.arguments.count > index
```

Comment:
I think this is off by one. If the count is 2, and the index is 1, that doesn't guarantee that there's an object at `arguments[2]`. I think this should be `ProcessInfo.processInfo.arguments.count > index + 1`

Reply:
Good catch 👍